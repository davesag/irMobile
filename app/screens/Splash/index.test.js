/**
 * @format
 */

import 'react-native'
import React from 'react'
import SplashScreen from '.'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

let tree

describe('rendering', () => {
  const navigation = {}

  beforeAll(() => {
    tree = renderer.create(<SplashScreen navigation={navigation} />)
  })

  it('rendered correctly', () => {
    expect(tree).toMatchSnapshot()
  })
})
