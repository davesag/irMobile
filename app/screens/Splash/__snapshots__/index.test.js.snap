// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering rendered correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#F8F9FA",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
      "marginBottom": 25,
      "top": 40,
    }
  }
>
  <Text>
    Independent Reserve
  </Text>
</View>
`;
